src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -18,7 +18,26 @@ public interface JsonApiMetricsConfig {
   @WithDefault("command")
   String command();
 
+  @NotBlank
+  @WithDefault("vector.enabled")
+  String vectorEnabled();
```

Comment:
Should this not be `Boolean`?

Reply:
This is provide Metrics tag name. Not the value.